src/App.test.js

```diff
@@ -7,6 +7,7 @@ describe('Wave 03: clicking like button and rendering App', () => {
     // Arrange
     const { container } = render(<App />);
     let buttons = container.querySelectorAll('button.like');
+    console.log(buttons);
```

Comment:
Remember to remove debugging print statements before opening a PR for review. We want to avoid print statements being committed to the code base to keep it readable and concise